src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyLineBreakAfter.java

```diff
@@ -26,10 +26,10 @@ else if (x < 0) {
                 else { break; }
                 switch (a)
                 {
-                case 0:
-                    break;
-                default:
-                    break;
+                    case 0:
```

Comment:
this looks like an unnecessary change. please undo.

Reply:
fixed